src/main/webapp/security/protected/admin/projectsManagement.xhtml

```diff
@@ -92,29 +93,30 @@
                                 </p:column>
 
                                 <p:column id="hdfsquotaColumn"
-                                          filterBy="#{projectsmanagement.hdfsQuota}"
-                                          sortBy="#{projectsmanagement.hdfsQuota}"
+                                          filterBy="#{projectsmanagement.getHdfsQuota(project.projectname)}"
+                                          sortBy="#{projectsmanagement.getHdfsQuota(project.projectname)}"
                                           filterMatchMode="contains"
                                           headerText="Disk quota"
                                           style=" font-size: 12px; text-align: center;">
-                                    <!--<p:cellEditor>-->
-                                        <!--<f:facet name="output">-->
+                                    <p:cellEditor>
+                                        <f:facet name="output">
                                             <h:outputText
-                                                value="#{projectsmanagement.hdfsQuota}" />
-                                        <!--</f:facet>-->
-                                        <!--<f:facet name="input">-->
-                                            <!--<p:inputText value="#{projectsmanagement.hdfsQuota}" style="width:100%"/>-->
-                                        <!--</f:facet>-->
-                                    <!--</p:cellEditor>-->
+                                                value="#{projectsmanagement.getHdfsQuota(project.projectname)}" />
+                                        </f:facet>
+                                        <f:facet name="input">
+                                            <p:inputText
+                                                value="#{projectsmanagement.hdfsquota}" style="width:100%"/>
```

Comment:
Validate that this is a valid number - .e.g, with: <f:convertNumber longOnly="true"/>